homework3/src/main/java/SmartDevice.java

```diff
@@ -0,0 +1,2 @@
+public interface SmartDevice {
```

Comment:
this interface does nothing. It should contain on/off and state checking functions at least.

Reply:
also, why doesn't it extend the ElectricityConsumer interface.